### ISSUE TITLE
terminate_execution is flaky

### ISSUE THREAD

**Issue by lucacasonato** [denoland/rusty_v8#710]
https://github.com/denoland/rusty_v8/pull/709/checks?check_run_id=2884761458

**Comment by bnoordhuis** [comment]
Can you paste the failure? I'm not seeing anything when I click on that link.

**Comment by lucacasonato** [comment]
```
failures:

---- terminate_execution stdout ----
Error: thread 'terminate_execution' panicked at 'called `Option::unwrap()` on a `None` value', src/context.rs:28:6


failures:
    terminate_execution

test result: FAILED. 111 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 130.31s

error: test failed, to rerun pass '--test test_api'
```

**Comment by bnoordhuis** [comment]
Fix in #736.
